frontend/packages/dev-console/src/const.ts

```diff
@@ -26,3 +26,9 @@ export const LAST_RESOURCE_TYPE_STORAGE_KEY = `devconsole.last.resource-type`;
 
 export const NAME_LABEL = 'app.kubernetes.io/name';
 export const INSTANCE_LABEL = 'app.kubernetes.io/instance';
+export const FLAG_DEVELOPER_CATALOG = 'DEVELOPER_CATALOG';
+export const FLAG_OPERATOR_BACKED_SERVICE_CATALOG_TYPE = 'OPERATOR_BACKED_SERVICE_CATALOG_TYPE';
+export const FLAG_SAMPLE_CATALOG_TYPE = 'SAMPLE_CATALOG_TYPE';
+export const OPERATOR_BACKED_SERVICE_SUB_CATALOG = 'OperatorBackedService';
+export const SAMPLE_SUB_CATALOG = 'Sample';
```

Comment:
Aren't these all catalog types? Why are some the suffix `_CATALOG_TYPE` and some `_SUB_CATALOG`?

These are catalog IDs, right? Can we add this to the names?

```tsx
export const OPERATOR_BACKED_SERVICE_CATALOG_TYPE_ID = 'OperatorBackedService';
export const SAMPLE_CATALOG_TYPE_ID = 'Sample';
```